tests/integration-tests/configs/common/common.yaml

```diff
@@ -263,6 +263,18 @@ networking:
   test_networking.py::test_public_private_network_topology:
     dimensions:
       - regions: ["af-south-1", "us-gov-east-1", "cn-northwest-1"]
+  test_cluster_networking.py::test_cluster_in_no_internet_subnet:
+    dimensions:
+        # The region needs to be the same of the Jenkins server since default pre/post install scripts are hosted in an
+        # S3 bucket belonging to the same region and S3 VPC Endpoints only work within the region.
+      - regions: ["us-east-1"]
```

Comment:
we should run these tests also in cn and govcloud. The configuration of VPC endpoints might require an update. This can be done as part of a separate PR

Reply:
Done